tests/baselines/reference/parenthesizedContexualTyping1.types

```diff
@@ -220,11 +220,11 @@ var l = fun(((Math.random() < 0.5 ? ((x => x)) : ((x => undefined)))), ((x => x)
 >x => undefined : (x: number) => any
 >x : number
 >undefined : undefined
->((x => x)) : (x: number) => number
->(x => x) : (x: number) => number
->x => x : (x: number) => number
->x : number
->x : number
+>((x => x)) : (x: any) => any
+>(x => x) : (x: any) => any
+>x => x : (x: any) => any
+>x : any
+>x : any

```

Comment:
This also seems wrong. The first two arguments should be skipped the first time around, and number is inferred from the third argument. Then we come back to visit all the lambdas, and T gets fixed to number. It should be fixed to number in all 3 lambdas, but looks like it only worked for the first two.